client/state/automated-transfer/selectors/is-automated-transfer-active.js

```diff
@@ -10,8 +10,15 @@ import 'calypso/state/automated-transfer/init';
  * @param {string} status name of current state in automated transfer
  * @returns {?boolean} is transfer currently active? null if unknown
  */
-export const isActive = ( status ) => ( status ? status === transferStates.START : null );
```

Comment:
We have several transitioning states (e.g. `provisioned`, `relocating_switcheroo`) that indicate that the transfer is in progress. I'm flipping this condition so that it won't break if we add more steps in the future.

Reply:
> If I were to be completely honest, I'm not a big fan of this implementation. The end goal is to abstract all this [read and poll transfer status] functionality and reuse it in https://github.com/Automattic/wp-calypso/pull/76949#issuecomment-1553376685.
> 
> But let's take one step at a time, I guess.

I have thought about this and it seem much effort to achieve at the moment, It seem we would have to convert the hosting page to functional component to get some reusable api that doesn't involve creating/modifying the set of redux actions. It would be easier with hooks.

Reply:
Further to whether this is still relevant, it seems this selector's change in behaviour is what's causing the unit test failure